### PR TITLE
Update philosophy.rst

### PR DESCRIPTION
**Pull request by spitGlued**
Small syntax corrections/changes to improve readability

### CODE REVIEW

**Comment by spitGlued** [comment]
Small syntax corrections/changes to improve readability

File: docs/philosophy.rst [hunk 1]
```diff
@@ -108,7 +108,7 @@ by *behave* as well as many other tools.
     then I should have three blue garments in stock
      and two black garments in stock.
 
-Each scenario is an exemplar, designed to illustrate a specific aspect of behavior of the application.
+Each scenario is an exemple, designed to illustrate a specific aspect of behavior of the application.
```
Comment (xrg) [inline docs/philosophy.rst]: That's actually worse (see wiktionary). An "exemplar" could fit, an "ex**a**mple" also.

Comment (spitGlued) [reply docs/philosophy.rst]: Thank's for the feedback.
I've reverted **exemple** back to **exemplar** in a later commit

**Comment by jenisys** [comment]
INTEGRATED INTO: behave-1.2.7.dev0 (with some tweaks).
Thx

**Pull request closed without merging**